### DISCUSSION TITLE
How to add my pipeline on the marketplace?

### DISCUSSION THREAD

**Discussion by Tavernari** [open-webui/open-webui#5503] [General]
Hi there,

I am studying pipelines, and after conducting some experiments, I developed one called o1-reasoning. It aims to add a sort of "chain of thought" feature to OpenWebUI.

Currently, it is in the alpha stage, as this is just the first implementation. I plan to use it in my daily work and increase its features over time.

You can find my repository where I host it at: [Tavernari/openwebui-pipelines](https://github.com/Tavernari/openwebui-pipelines).

How can I add this pipeline to [openwebui.com](https://openwebui.com/)?

**Comment by GrayXu** [comment]
You can create a new one in your profile after registering an account.
